prac_06/used_cars.py

```diff
@@ -11,8 +11,17 @@ def main():
     """Demo test code to show how to use car class."""
     my_car = Car(180)
     my_car.drive(30)
+    my_car.name = "My car"
     print(f"Car has fuel: {my_car.fuel}")
     print(my_car)
 
+    limo = Car(100)
```

Comment:
Same as with my_car can remove limo.name by using the line and adding its name then fuel